Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -584,9 +584,11 @@ void AXObjectCache::setIsolatedTreeFocusedObject(AccessibilityObject* focus)
 AccessibilityObject* AXObjectCache::get(Node& node) const
 {
     auto* renderer = node.renderer();
-    auto renderID = renderer ? m_renderObjectMapping.getOptional(*renderer) : std::nullopt;
-    if (renderID)
-        return m_objects.get(*renderID);
+    if (renderer && !renderer->isYouTubeReplacement()) {
+        auto renderID = renderer ? m_renderObjectMapping.getOptional(*renderer) : std::nullopt;
```

Comment:
I think this `renderer ?` check is unnecessary since we check that it's non-null to enter this block.